infrastructure/prime-app-template.yml

```diff
@@ -999,9 +977,24 @@ objects:
           - name: ${SVC_NAME}-redis-data
             emptyDir: {}
         containers:
-          - resources:
+          - name: redis
+            image: >-
+              image-registry.openshift-image-registry.svc:5000/${OC_LICENSE_PLATE}-tools/redis-6:latest
```

Comment:
Latest?

Reply:
yes, it's an image in our tools namespace.

Reply:
Ah I see, we control it what version is in there.